### DISCUSSION TITLE
v9.4.0 gives 404 for static pages when pkg-ed

### DISCUSSION THREAD

**Discussion by nikoskleidis** [vercel/next.js#13343] [Help]
# Bug report

## Describe the bug

I use [pkg](https://github.com/zeit/pkg) to package my nextjs app. 
When I upgraded to version 9.4.0 from 9.3.6 the static pages I tried to access were all redirecting to the 404 page. 
SSR pages work as expected on 9.4.0.
Also everything works as expected on dev mode

## To Reproduce

1. Go to this [repo](https://github.com/nikoskleidis/pkg-test)
2. On master branch run yarn install, yarn build, yarn pkg (or pkg-macos)
3. Execute `./pkg/test` to run the app in the self contained server
4. Notice that we cannot access the localhost:3000 or the localhost:3000/static page but we can access the localhost:3000/ssr which is a server side rendered page

## Expected behavior

The expected behavior is that all pages (localhost:3000, localhost:3000/static, localhost:3000/ssr) render normally.

## System information

- OS: [macOS, linux]
- Version of Next.js: [9.4.0]
- Version of Node.js: [10]

## Additional context

If you checkout branch **9.3.6** which has the previous version of nextjs and perform the same steps you will notice that everything works as expected

**Comment by Bleuzen** [comment]
Having the same issue. Did you find any solution yet?

**Reply by nikoskleidis** [reply]
Yes I did. Check this issue https://github.com/vercel/pkg/issues/917
